### PR TITLE
Add FCL header to fcl-js index

### LINKED ISSUE #444: Create FCL Landing Page
Very similar to the cadence landing page (/cadence) but for FCL. Utilize the new FCL Logo (same one used in repo switcher) and put in placeholder links for now.

### PR DESCRIPTION
**Pull request by aureentuluvacal**
Closes #444

### CODE REVIEW

File: app/ui/design-system/src/lib/Pages/InternalPage/index.tsx [hunk 1]
```diff
@@ -96,10 +96,15 @@ export function InternalPage({
   const { pathname } = useLocation()
 
   useEffect(() => {
-    if (contentRef.current) {
+    if (
+      contentRef.current &&
+      !["cadence", "fcl-js"].includes(contentPath) &&
```
Comment (joekrill) [inline app/ui/design-system/src/lib/Pages/InternalPage/index.tsx:101]: Could we just check if the `header`  prop here is defined?

File: app/constants/repos/contents-structure.ts [hunk 2]
```diff
@@ -122,6 +123,7 @@ export const ROUTING_STRUCTURE: Partial<
   learn: ["kitty-items", "concepts"],
   community: ["bounties"],
   cadence: ["language", "tutorial"],
+  "fcl-js": ["tutorial"],
```
[the remark this answers is not in the document]
Comment (aureentuluvacal) [reply app/constants/repos/contents-structure.ts:126]: It loads the same content right? I think it makes sense to keep it. Well it's arbitrary to me 😂

File: app/ui/design-system/src/lib/Pages/InternalPage/index.tsx (hunk 1 above)
Comment (aureentuluvacal) [reply app/ui/design-system/src/lib/Pages/InternalPage/index.tsx:101]: Omg that's so much nicer.

File: app/constants/repos/contents-structure.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (joekrill) [reply app/constants/repos/contents-structure.ts:126]: I guess I'd leave that up to @10thfloor and @srinjoyc, then!

**Pull request merged**